typescript/src/httpHandlerFactory.ts

```diff
@@ -0,0 +1,47 @@
+import { HttpHandler, IHttpHandler } from "./httpHandler";
+
+function getPriorityBaseUrl(): string {
+  if (process.env.ORCHESTRATE_BASE_URL) {
+    return process.env.ORCHESTRATE_BASE_URL;
+  }
+  return "https://api.careevolutionapi.com";
+}
+
+function getPriorityApiKey(apiKey: string | undefined): string | undefined {
+  if (apiKey) {
```

Comment:
I think this whole method could be 

`return apiKey || process.env.ORCHESTRATE_API_KEY`